_gsocproposals/2023/proposal_GangaSnakemake.md

```diff
@@ -0,0 +1,43 @@
+---
+project: Ganga
+title: A concurrency model for the monitoring in Ganga
+layout: gsoc_proposal
+year: 2023
+difficulty: medium
+duration: 350
+mentor_avail: June-August
```

Comment:
That availability is not exactly compatible with the project duration of 350 hours. Can you clarify?

Reply:
Too quick updating header. Updated availability and in fact also the title inside the file.